themes/bootstrap3/templates/cart/export.phtml

```diff
@@ -61,10 +61,16 @@
   $('#format').change(function exportFormatChange(e) {
     if (this.selectedOptions[0].getAttribute('data-redirect') === null) {
       $('.export.btn').removeAttr('data-lightbox-ignore');
+      $('.export.btn').unbind('click');
+      $('form[name=exportForm]').removeAttr('target');
```

Comment:
Are you sure we want to remove the target from the form? I have a feeling this might cause Lightbox problems.